code/_onclick/hud/new_player.dm

```diff
@@ -1,4 +1,11 @@
+#define SHUTTER_MOVEMENT_DURATION 0.4 SECONDS
+#define SHUTTER_WAIT_DURATION 0.2 SECONDS
+
 /datum/hud/new_player
+	///Whether the menu is currently on the client's screen or not
+	var/menu_hud_status = TRUE
+	///List of menu HUD elements that we hide when collapsing the menu
+	var/list/toggleable_elements = list()
```

Comment:
what happens if one of these elements is deleted without clearing the hud? will this hang a ref and cause a harddel?

Reply:
would signal usage be better here maybe?

Reply:
fuck it we ball https://github.com/tgstation/tgstation/pull/76443/commits/7d1dbbcb8e604ae8d176707862cb563c7b32674f